diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index 90b206e..5ee98cb 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -23,19 +23,32 @@ plugins {
 }
 
 android {
+    // Existing configurations
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
+    
+    // Define the new configuration
+    configurations {
+        resolvedImplementation {
+            canBeResolved = true
+            canBeConsumed = false
+            extendsFrom implementation
+        }
+    }
+
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
         targetSdkVersion safeExtGet('targetSdkVersion', 28)
         versionCode 1
         versionName "1.0"
     }
+
     lintOptions {
         abortOnError false
     }
 }
 
+
 repositories {
     mavenLocal()
     maven {
@@ -59,11 +72,14 @@ dependencies {
 afterEvaluate { project ->
 
     task androidJavadoc(type: Javadoc) {
-        source = android.sourceSets.main.java.srcDirs
-        classpath += files(android.bootClasspath)
-        classpath += files(project.getConfigurations().getByName('compile').asList())
-        include '**/*.java'
-    }
+    source = android.sourceSets.main.java.srcDirs
+    classpath += files(android.bootClasspath)
+
+    // Use the new resolvable configuration for classpath
+    classpath += files(project.configurations.resolvedImplementation)
+    include '**/*.java'
+}
+
 
     task androidJavadocJar(type: Jar, dependsOn: androidJavadoc) {
         archiveClassifier = 'javadoc'
